modules/navbar-action.js

```diff
@@ -0,0 +1,21 @@
+import show from './show.js';
+import createBookList from './create-book-list.js';
+
+const btnHome = document.getElementById('navbtn-home');
+const btnList = document.getElementById('navbtn-list');
+const btnAdd = document.getElementById('navbtn-add');
+const btnContact = document.getElementById('navbtn-contact');
+
+btnHome.addEventListener('click', () => {
+  show('home');
+});
+export default btnList.addEventListener('click', () => {
+  createBookList();
+  show('list');
+});
+btnAdd.addEventListener('click', () => {
+  show('add');
+});
+btnContact.addEventListener('click', () => {
+  show('contact');
+});
```

Comment:
- [ ] Kind remainder let's make sure we have the time displayed as per requirements just below our navbar that currently is not implemented 
![image](https://user-images.githubusercontent.com/87689547/159906717-ddde43da-9e4d-42de-9688-dd19575c211e.png)